src/app/public/modules/country-field/types/country.ts

```diff
@@ -0,0 +1,18 @@
+export interface SkyCountryFieldCountry {
+
+  /**
+   * The name of the country
+   * @required
+   */
+  name: string;
+
+  /**
+   *  The [International Organization for Standardization Alpha 2](https://www.nationsonline.org/oneworld/country_code_list.htm)
+   * country code for the country
```

Comment:
It's not a complete sentence, but I'd still end with a period here.

Reply:
Done